src/api.ts

```diff
@@ -142,6 +142,12 @@ export default class Chromeless<T extends any> implements Promise<T> {
     return this
   }
 
+  setDocumentContent(html: string): Chromeless<T> {
```

Comment:
I actually do think there should be a `getHtml()` method. That way, the example usage for `setHtml()` could look more like this:

```js
const html = await chromeless
  .setHtml('<h1>Hello world!</h1>')
  .getHtml()

console.log(html) // returns <html><head></head><body><h1>Hello world!</h1></body></html>
```